estuary_updater/handlers/errata.py

```diff
@@ -0,0 +1,89 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+from estuary_updater.handlers.base import BaseHandler
+from estuary.models.errata import Advisory
+from estuary.models.bugzilla import BugzillaBug
+from estuary.utils.general import timestamp_to_datetime
+
+import requests
+import requests_kerberos
+
+
+class ErrataHandler(BaseHandler):
+    """A handler for dist-git related messages."""
+
+    @staticmethod
+    def can_handle(msg):
+        """
+        Determine if this handler can handle this message.
+
+        :param dict msg: a message to be analyzed
+        :return: a bool based on if the handler can handle this kind of message
+        :rtype: bool
+        """
+        return msg['topic'] == '/topic/VirtualTopic.eng.errata.activity.status'
+
+    def handle(self, msg):
+        """
+        Handle an Errata tool message and update Neo4j if necessary.
+
+        :param dict msg: a message to be processed
+        """
+        advisory_id = msg['body']['headers']['errata_id']
+
+        url = '{0}/erratum/{1}'.format(
+            self.config['estuary_updater.errata_api_url'].rstrip('/'), advisory_id)
+        response = requests.get(url, auth=requests_kerberos.HTTPKerberosAuth())
+        advisory_json = response.json()
+
+        advisory_type = msg['body']['headers']['type'].lower()
+        advisory_info = advisory_json['errata'][advisory_type]
+        # TODO: The way DateTime objects are handled will need to be changed once we stop using the
+        # API, as the strings are formatted differently in the message bus messages
+        actual_ship_date = advisory_info['actual_ship_date']
+        if actual_ship_date:
```

Comment:
I haven't tested this but I think you can convert all these if statements to the following:
```python
advisory = {
    'advisory_name': msg['body']['msg']['fulladvisory'],
    'content_types': advisory_json['errata'][advisory_type]['content_types'],
    'id_': advisory_id,
    'product_short_name': msg['body']['msg']['product'],
    'security_impact': advisory_json['errata'][advisory_type]['security_impact'],
    'security_sla': advisory_json['errata'][advisory_type]['security_sla'],
    'synopsis': msg['body']['headers']['synopsis']
}
for dt in ('actual_ship_date', 'created_at', 'issue_date', 'release_date', 'status_updated_at', 'update_date'):
    if advisory_json['errata'][advisory_type][dt]:
        if dt == 'status_updated_at':
            advisory[dt] = advisory_json['errata'][advisory_type][dt]
        else:
            advisory['status_time'] = advisory_json['errata'][advisory_type][dt]
```